core/api/src/graphql/public/types/object/quiz.ts

```diff
@@ -1,5 +1,7 @@
 import SatAmount from "../../../shared/types/scalar/sat-amount"
 
+import Timestamp from "@/graphql/shared/types/scalar/timestamp"
```

Comment:
use the same import format in all the imports `"@/graphql/`

Reply:
you mean for SatAmount?

Reply:
yes

Reply:
yeah but that's almost another refactoring PR :)